packages/libs/react-ui/src/components/Pagination/Pagination.stories.tsx

```diff
@@ -65,6 +74,7 @@ export const Controlled: Story = {
           currentPage={page}
           label={label}
           visiblePageLimit={visiblePageLimit}
+          initialSetPage={initialSetPage}
```

Comment:
nit: Do you think `initialPage` would be more concise and still clear?

Reply:
Should have been _initialSelectedPage_ , but _initialPage_ is fine too

Reply:
initialSelectedPage is fine with me as well! `initialSetPage` just felt a bit strange, but I'm ok with either of the other options 👍

Reply:
Set to _initialSelectedPage_